src/testHelpers.ts

```diff
@@ -1,16 +1,57 @@
+import { sha256, toBytes } from 'viem';
 import { privateKeyToAccount, PrivateKeyAccount } from 'viem/accounts';
 import { config } from 'dotenv';
-
 config();
 
-export function getTestPrivateKeyAccount(): PrivateKeyAccount {
-  const privateKey = process.env.PRIVATE_KEY;
+// Source: https://github.com/OffchainLabs/nitro-testnode/blob/master/scripts/accounts.ts#L28
+type NitroTestNodePrivateKeyAccounts = {
+  // funnel
+  deployer: PrivateKeyAccount & { privateKey: `0x${string}` };
+  // sequencer (batch poster and rollup owner are the same in nitro-testnode)
+  l2RollupOwner: PrivateKeyAccount & { privateKey: `0x${string}` };
+  // l3owner
+  l3RollupOwner: PrivateKeyAccount & { privateKey: `0x${string}` };
+  // sha256(user_token_bridge_deployer)
+  l3TokenBridgeDeployer: PrivateKeyAccount & { privateKey: `0x${string}` };
+};
 
-  if (typeof privateKey === 'undefined') {
-    throw Error(`missing PRIVATE_KEY env variable`);
+export function getNitroTestnodePrivateKeyAccounts(): NitroTestNodePrivateKeyAccounts {
```

Comment:
Modified this function to get all private keys from the .env file, and generate the one for the deployer of the canonical TokenBridge (needed because that's the account that holds all native tokens, and we need to transfer some to the creator of the new token bridge)